src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -138,13 +131,28 @@ class DocumentStatisticsTrainingOnlyPolicy {
     // ----------------------------------------------------------------------
     static constexpr char const * const     NameValue = DocumentStatisticsEstimatorName;
 
+    enum AnalyzerMethod {
+        WORD = 0,
```

Comment:
>0 [](start = 15, length = 1)

The first value should be 1 to avoid potential problems with languages that default initialize to 0.